src/pages/home/component.jsx

```diff
@@ -1,10 +1,65 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './styles.scss';
 
+import Expand from 'components/expand';
+
+function ChooseCountry() {
+  return (
+    <div className="c-choose-country">
+      <button>Canada</button>
+      <button>Indonesia</button>
+      <button>Sweden</button>
+      <button>Peru</button>
+      <button>Tanzania</button>
+    </div>
+  );
+}
+
 function HomePage() {
+  const sections = [
+    {
+      title: 'See species distribution.',
+      body:
+        'Our i whales Dry place replenish darkness had fifth very subdue in midst behold face two. That winged evening fourth.'
+    },
+    {
+      title: 'See bioclimatic variables.',
+      body:
+        'Our i whales Dry place replenish darkness had fifth very subdue in midst behold face two. That winged evening fourth.'
+    }
+  ];
+  const [sectionIndex, setSection] = useState(0);
+  const [transition, setTransition] = useState('fade-in');
+  const section = sections[sectionIndex];
+
+  const changeSection = next => {
+    setTransition('fade-out');
+    setTimeout(() => setSection(next), 200);
+    setTimeout(() => setTransition('fade-in'), 300);
```

Comment:
I dont love this, why not paint both and toggle the opacity of the active one and the inactive one. Or if that's too complicated, why not use `react-spring`? If you want, we can talk about this offline.

Reply:
Also you're not clearing these timeouts